src/burn/drv/irem/d_m107.cpp

```diff
@@ -1023,7 +1023,7 @@ static void scanline_interrupts(INT32 scanline)
 
 		if (scanline>=8 && scanline < 248 && nPrevScreenPos != (scanline-8)+1) {
 			if (nPrevScreenPos >= 0 && nPrevScreenPos <= 239)
-				DrawLayers(nPrevScreenPos, (scanline-8)+1);
+			//	DrawLayers(nPrevScreenPos, (scanline-8)+1);
```

Comment:
Are you sure about this one ? `nPrevScreenPos = (scanline-8)+1;` becomes conditional if you do this.

Reply:
I'll be honest the changes were done by a previous dev called +T+ originally for the m92
driver as these games were painfully slow on the xbox when using FBA, i just copied this
for the m107 driver but if you have a better idea of how this should be implemented
then i'd sure like to hear about it.

Certainly it's a feature of the drivers you can do this in MAME as well although in older
pre MAME119 builds you dont have to as the grahical handling is less resource intensive
anyway we've had the m92 and m107 drivesr setup this way for years now with not one
single report of any issues so i do think it's safe to do this.

Just a heads up though dont disable the scalines on the drawlayer in the M72 driver
as it will break the half of the games graphically.